redbadger/react.london
# Select from events based upon their displayLevel value

![candles](https://cloud.githubusercontent.com/assets/6134406/17548606/8763e57c-5ee4-11e6-90f1-87809984d2a3.gif)



- [x] Remove ticket container
- [x] Pull featuredEvent from events by looking for an event with the `"Featured"` display level

## Notes

The site will look the same and display the same information as before. From the user's point of view nothing has changed.

## Test plan

- Remove the `featuredEvent` from React London. The site will still render the same as this property is no longer used.
- Set all `displayLevel` properties on RL events to be not `Featured`. The site will render successfully without information as no event has been picked to display.
- Set one RL event `displayLevel` to `Featured`. This one will be displayed by the site.
